students/mariusz_michalczyk/lesson_03_functions/Negative_exponent.py

```diff
@@ -0,0 +1,7 @@
+def calculate(a, n):
+    return a ** n
```

Comment:
I guess what they meant here is to add another `elif` to `exponent.py` and do a division there.
```
def power(a, n):
   if n == 0:
       return 1
   elif n > 0:
       return a * power(a, n - 1)
   else:
       return power(a, n + 1) / a
```